aws/src/main/python/beam_lambda/lambda_function.py

```diff
@@ -607,6 +671,10 @@ def stop_instance(instance_ids):
 def terminate_instance(instance_ids):
     return ec2.terminate_instances(InstanceIds=instance_ids)
 
+def calculate_max_ram(instance_type):
```

Comment:
This isn't horrible, but a possible simplified solution would be:

```
  max_system_ram = 15
  percent_towards_system_ram = .25
  return ram - min(ram * percent_towards_system_ram, max_system_ram)
```

where 15 could be turned into a variable

Reply:
@JustinPihony is it fine to run large instances on 15 GB like 2000GB or 4000GB . because we have instances having this much of memory.

Reply:
That is more than the current calculation - 15 gb versus (4000*.0025=10). So, I figured it was about what was expected overall. As to if that is enough system memory left - that is something we can tweak as needed.

Reply:
thanks @JustinPihony , Change max ram calculation as per suggestion